modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/source/components/ShaftConnectedCompressor1Phase.md

```diff
@@ -34,7 +32,7 @@ S^{\text{momentum}} = \Delta p A_{\text{ref}} \cdot \hat{n}_{\text{out}},
 and
 
 !equation id=energy_source
-S^{\text{energy}} = -(\tau_{\text{isentropic}} + \tau_{\text{dissipation}} + \tau_{\text{friction}}) \omega,
+S^{\text{energy}} = -(\tau_{\text{isentropic}} + \tau_{\text{dissipation}}) \omega,
```

Comment:
Did you read #20206? As far as I can tell, RELAP5 doesn't even have a friction torque for their compressor - they have it for one of their pumps. I think it was just copied from our existing components assuming it would still be there. Plus, note my comment about the design: a more flexible design would separate the friction torque into a separate component (`ShaftConnectedMotor`), rather than force their 4-coefficient model, since there is no coupling to the flow itself, just shaft speed.

Reply:
I'll check with the RELAP5 team, but I think their compressor component does have friction torque. It does have input coefficients for it that are described in the input manual. Anyway, it is much more user friendly to have the friction torque in the component itself, you just set the coefficients instead of creating the function and the motor component.

Reply:
> I'll check with the RELAP5 team, but I think their compressor component does have friction torque. It does have input coefficients for it that are described in the input manual.

Ok, it's not in their compressor section in their manual (Vol. I), but maybe it's in their model anyway. I'll put the friction torque back (but only on the shaft).

Reply:
Yeah, I went checking their Vol II, appendix A. They don't have that "shaft drive" section for the compressor as they have for the pump in the theory manual. Probably because it is the same. I've checked with George, and they do have it.